build.gradle

```diff
@@ -211,6 +211,14 @@ subprojects {
             csv.enabled = false
         }
     }
+
+    if (!path.startsWith(':cloudfoundry-identity-samples')) {
+        task collectDependencies(type: Copy) {
+            from configurations.default
```

Comment:
What is gradle default configuration? We only need runtime for collecting dependencies, right?

Reply:
Hmm, `default` == `runtime` if not customized. I can try switching to runtime.